.github/CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
-* Bug reports and feature requests are welcome in [the issues][issues]
-* Pull Requests are welcome. For more complex changes and features it's
-  recommended to open an issue with the feature request first
-* Documentation contributions are as important as code contributions
+## You can find our [contribution guide on our website](https://taskfile.dev/contributing)
+
+- Please read it carefully before opening a PR.
+- If you have any questions, you can:
+  - [Open an issue][issues]
+  - [Create a discussion][discussions]
+  - [Chat to us on Discord][discord]
 
```

Comment:
Here may need a`<!-- prettier-ignore-start -->`, as is the bottom.

Reply:
Prettier seems to be ignoring any files inside the `.github` folder, so this isn't strictly needed, but I'll add it to be consistent. Maybe we can make prettier work with these files in the future. 

I've also changed the contribution guide link to the same style 👍